services/requests/feasibility_constraint_definitions/schemas.go

```diff
@@ -0,0 +1,120 @@
+package feasibility_constraint_definitions
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/leanspace/terraform-provider-leanspace/helper"
+	"github.com/leanspace/terraform-provider-leanspace/helper/general_objects"
+)
+
+var feasibilityConstraintDefinitionSchema = map[string]*schema.Schema{
+	"id": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"name": {
+		Type:     schema.TypeString,
+		Required: true,
+	},
+	"description": {
+		Type:     schema.TypeString,
+		Optional: true,
+	},
+	"cloned": {
+		Type:     schema.TypeBool,
+		Optional: true,
+	},
+	"argument_definitions": {
+		Type:     schema.TypeSet,
+		Optional: true,
+		MaxItems: 499,
+		Elem: &schema.Resource{
+			Schema: argumentDefinitionSchema,
+		},
+	},
+	"created_at": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"created_by": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"last_modified_at": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"last_modified_by": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+}
+
+var argumentDefinitionSchema = map[string]*schema.Schema{
+	"name": {
+		Type:     schema.TypeString,
+		Required: true,
+	},
+	"description": {
+		Type:     schema.TypeString,
+		Optional: true,
+	},
+	"attributes": {
+		Type:     schema.TypeList,
+		Required: true,
+		MinItems: 1,
+		MaxItems: 1,
+		Elem: &schema.Resource{
+			Schema: general_objects.DefinitionAttributeSchema(
+				[]string{"TEXT", "BINARY", "BOOLEAN", "ENUM", "TIMESTAMP", "DATE", "ARRAY", "STRUCTURE", "TLE"}, // attribute types not allowed in command definition attributes
+				nil,   // All fields are used
+				false, // Does not force recreation if the type changes
+			),
+		},
+	},
+}
+
+var feasibilityConstraintDefinitionFilterSchema = map[string]*schema.Schema{
+	"ids": {
+		Type:     schema.TypeList,
+		Optional: true,
+		Elem: &schema.Schema{
+			Type:         schema.TypeString,
+			ValidateFunc: validation.IsUUID,
+		},
+	},
+	"query": {
+		Type:     schema.TypeString,
+		Optional: true,
+	},
```

Comment:
these are already defined [here](https://github.com/leanspace/terraform-provider-leanspace/blob/main/helper/general_objects/schemas.go#L87-L97)

Reply:
removed, I did not find how it was injected into the schema (if you can present it to me :)  ) so I added them to be sure

Reply:
it's [this function](https://github.com/leanspace/terraform-provider-leanspace/blob/main/helper/general_objects/schemas.go#L85C6-L85C18), we take the keys defined in the resource's schema and add it to the base ones

Reply:
okay so when we declare a filterSchema, that baseFilter is automatically applied